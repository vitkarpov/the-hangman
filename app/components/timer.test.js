const Timer = require("./timer");

describe("Timer", () => {
  it("renders correctly", () => {
    expect(new Timer().render()).toMatchSnapshot();
  });

  it("renders correctly after a second pass", () => {
    const node = document.createElement("div");
    node.innerHTML = new Timer().render();

    setTimeout(() => {
      expect(node.querySelector(".timer").innerText).toBe("1");
    }, 1000);
  });
});
