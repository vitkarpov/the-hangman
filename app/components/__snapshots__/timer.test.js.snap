// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer renders correctly 1`] = `"<span class=\\"timer\\">0</span>"`;
